examples/tasks/complex-example.yaml

```diff
@@ -0,0 +1,55 @@
+--- # The Header starts after this line
+  version: 1        # All tasks, like plugins, have versioning
+  schedule:         # How frequently should we run this task?
+    type: "simple"  # Simple means run forever at the interval below
+    interval: "1s"  # This accepts seconds ('s') and milliseconds  ('ms')
+  workflow:
+   collect:
+     metrics:
+       /intel/pcm/EXEC: {}
+       /intel/pcm/FREQ: {}
+       /intel/pcm/INST: {}
+       /intel/pcm/INSTnom: {}
+       /intel/pcm/INSTnom%: {}
+       /intel/psutil/load/load1: {}
+       /intel/psutil/load/load5: {}
+       /intel/psutil/vm/available: {}
+       /intel/psutil/vm/free: {}
+       /intel/psutil/vm/used: {}
+       /intel/haproxy/info/Idle_pct: {}
+       /intel/haproxy/info/MaxConnRate: {}
+       /intel/haproxy/info/MaxSessRate: {}
+       /intel/haproxy/info/MaxSslConns: {}
+       /intel/linux/docker/cpu_stats/cpu_usage/total_usage: {}
+       /intel/linux/docker/cpu_stats/cpu_usage/usage_in_kernelmode: {}
+       /intel/linux/docker/cpu_stats/cpu_usage/usage_in_usermode: {}
+       /intel/linux/docker/cpu_stats/throttling_data/periods: {}
+       /intel/linux/docker/cpu_stats/throttling_data/throttled_periods: {}
+       /intel/linux/docker/cpu_stats/throttling_data/throttled_time: {}
+     config:
+        /intel:
```

Comment:
That's a great point. I'm guilty of continuing that practice as I've seen it in example tasks since day one. Let's be more accountable for that across all examples @intelsdi-x/snap-maintainers